js/script.js

```diff
@@ -0,0 +1,94 @@
+const mapLink = document.querySelector('.contacts-map-link');
+const mapModal = document.querySelector('.modal-map');
+const feedbackLink = document.querySelector('.feedback-link');
+const feedbackModal = document.querySelector('.modal-feedback');
+const feedbackForm = feedbackModal.querySelector('.feedback-form');
+const feedbackName = feedbackModal.querySelector('#feedback-name');
+const feedbackEmail = feedbackModal.querySelector('#feedback-email');
+const feedbackMessage = feedbackModal.querySelector('#feedback-message');
+const promoSlide = document.querySelectorAll('.promo-slider-item');
+const promoSliderButton = document.querySelectorAll('.promo-slider-controls-button');
+const servicesSlide = document.querySelectorAll('.services-slider-item');
+const servicesSliderButton = document.querySelectorAll('.services-controls-button');
```

Comment:
```suggestion
const promoSlides = document.querySelectorAll('.promo-slider-item');
const promoSliderButtons = document.querySelectorAll('.promo-slider-controls-button');
const servicesSlides = document.querySelectorAll('.services-slider-item');
const servicesSliderButtons = document.querySelectorAll('.services-controls-button');
```

так как метод `querySelectorAll()` выбирает коллекцию элементов.